addons/account/views/res_company_views.xml

```diff
@@ -33,4 +33,36 @@
         </field>
     </record>
 
+    <!-- Onboarding -->
+    <record id="res_company_form_view_onboarding" model="ir.ui.view">
+        <field name="name">res.company.form.view.onboarding</field>
+        <field name="model">res.company</field>
+        <field name="inherit_id" ref="base.view_company_form"/>
+        <field name="mode">primary</field>
+        <field name="priority">1000</field>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='social_media']" position="replace"/>
+            <form position="attributes">
+                <attribute name="js_class">company_onboarding_form</attribute>
+            </form>
+        </field>
+    </record>
+
+    <record id="res_company_sale_tax_form_view_onboarding" model="ir.ui.view">
+        <field name="name">account.onboarding.sale.tax.form</field>
+        <field name="model">res.company</field>
+        <field name="mode">primary</field>
```

Comment:
Note that this is not necessary to give it if you don't inherit from another view.